src/lib/graveyard-container.js

```diff
@@ -0,0 +1,19 @@
+class Graveyard {
+  constructor(refObject) {
+    refObject.onInserted.add(this.onInserted);
+  }
+
+  onInserted(container, insertedObjects, player) {
+    for (let i = 0; i < insertedObjects.length; i++) {
+      const name = insertedObjects[i].getTemplateName()
+      const playerName = player.getName()
+      if(container.remove(insertedObjects[i])) {
```

Comment:
In TTPG you can find the destination bag and call dstBag.addObjects(insertedObj) without first removing it from the graveyard bag -- TTPG takes care of the transfer.

Reply:
Cheers for letting me know about that. This removal is a placeholder that I'll replace with something like a switch statement that calls different utility functions depending on the type of object